README.md

```diff
@@ -1,2 +1,87 @@
-# Street Ends mobile 
+# :evergreen_tree: Street Ends Mobile 
 Shoreline Access for Everyone
+
+Welcome to the Street Ends mobile app Github. The Steet Ends app aims to offer you an easy and convenient resource for finding respite in the public parks that dot Seattle's waterfront, and discover the story behind the people and communities who make it all possible. This app is a work-in-progress, built using React Native for cross platform support, made by Loan Pham, David Prismantas, Alex Sutherland, Jennifer Villacis, and Andy Wilson.
+
+## :computer: Local Deployment Instructions
+
+
+### :apple:  Deployment on a Mac
+
+
+### :hammer:  Setting Up Dev Environment
+
+To properly set up your local machine for React Native development, you will need the following:
```

Comment:
I think you can just reference the RN setup instructions & make clear developers should use the CLI set for their platform - https://reactnative.dev/docs/environment-setup

Reply:
Haha, I thought about that but I figured you would want more effort put in than just providing the link. I pushed a new update with the reference to the official docs and Jen's suggestion.